tools/mcsema_disass/ida7/get_cfg.py

```diff
@@ -470,7 +470,7 @@ def is_start_of_function(ea):
   if not is_code(ea):
     return False
 
-  name = idc.get_func_name(ea) or idc.GetTrueName(ea)
+  name = idc.GetTrueName(ea) or idc.get_func_name(ea)
```

Comment:
@kumarak is a corresponding change needed in IDA 6.xx code?

Reply:
I don't see an issue with IDA 6.xx. `idc.GetFunctionName` and `idc.GetTrueName` returns the same name for entrypoint.

Reply:
What I mean is, can you apply the same order in the IDA 6.xx code?